tests/mine.rs

```diff
@@ -1262,3 +1262,9 @@ fn minivec_assume_minivec_init() {
   assert_eq!(bytes[0], 137);
   assert_eq!(bytes[511], 137);
 }
+
+#[test]
+fn minivec_should_fail_try_reserve_impossible() {
+  let mut buf = minivec::mini_vec![0; 512];
+  buf.try_reserve(usize::max_value()).expect_err("FAIL");
+}
```

Comment:
This test is incomplete.

We've tested failure which is good but we need to test the success arm as well.

I also don't see `try_reserve_exact()` here either.

Reply:
https://github.com/DoumanAsh/minivec/blob/try_reserve/src/lib.rs#L1172

Reply:
This is good but we also need to comment back in the tests in `vec.rs` as well. Seems like there's a good bit of tests the stdlib has there.